### PR TITLE
mp3directcut: Add version 2.25

### PR DESCRIPTION
**Pull request by digrouz**
Hello,

After first run, the process create mp3DirectCut.ini file that could be persistable. I tried with the persist feature but it creates it as a folder, Any idea how to implement it properly ?

### CODE REVIEW

**Comment by digrouz** [comment]
the "mp3DirectCut.ini" item created by the persist is a folder shortcut, any idea why ?

**Comment by Ash258** [comment]
No it is not with my changes. See `pre_install` (L12-14)

File: bucket/mp3directcut.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    "homepage": "http://mpesch3.de",
+    "license": "Freeware",
+    "url": "https://www.fosshub.com/mp3DirectCut.html/mp3DC225.exe",
+    "hash": "md5:872f3923af1982fc39a18d724ffd5904",
+    "installer": {
+        "script": [
+            "Expand-7zipArchive \"$dir\\$fname\" -Removal",
+            "if (-not (Test-Path \"$persist_dir\\mp3DirectCut.ini\")) {",
+            "    Set-Content \"$dir\\mp3DirectCut.ini\" '[mp3DirectCut]' | Out-Null",
+            "}"
```
Comment (linsui) [inline bucket/mp3directcut.json]:
```suggestion
            "    Set-Content \"$dir\\mp3DirectCut.ini\" @('[mp3DirectCut]', \"version=$version\") | Out-Null",
            "} else {",
            "    (Get-Content \"$persist_dir\\mp3DirectCut.ini\") -replace 'version=.+', \"version=$version\" | Set-Content \"$persist_dir\\mp3DirectCut.ini\"",
            "}",
            "New-Item \"$dir\\mp3DClocal.ini\" | Out-Null"
```
Set version to disable welcome window. mp3DClocal.ini is not really needed. It seems that without it portable mode is still enabled.

File: bucket/mp3directcut.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    "description": "Lossless audio editor and recorder for MP3 files.",
+    "homepage": "http://mpesch3.de",
+    "license": "Freeware",
+    "url": "https://www.fosshub.com/mp3DirectCut.html/mp3DC225.exe",
+    "hash": "md5:872f3923af1982fc39a18d724ffd5904",
+    "installer": {
+        "script": [
+            "Expand-7zipArchive \"$dir\\$fname\" -Removal",
+            "if (-not (Test-Path \"$persist_dir\\mp3DirectCut.ini\")) {",
+            "    Set-Content \"$dir\\mp3DirectCut.ini\" '[mp3DirectCut]' | Out-Null",
```
Comment (linsui) [inline bucket/mp3directcut.json]:
```suggestion
```

**Comment by digrouz** [comment]
everything seems good to me

File: bucket/mp3directcut.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    "description": "Lossless audio editor and recorder for MP3 files.",
+    "homepage": "http://mpesch3.de",
+    "license": "Freeware",
+    "url": "https://www.fosshub.com/mp3DirectCut.html/mp3DC225.exe",
+    "hash": "md5:872f3923af1982fc39a18d724ffd5904",
+    "installer": {
+        "script": [
+            "Expand-7zipArchive \"$dir\\$fname\" -Removal",
+            "if (-not (Test-Path \"$persist_dir\\mp3DirectCut.ini\")) {",
+            "    Set-Content \"$dir\\mp3DirectCut.ini\" @('[mp3DirectCut]', \"version=$version\") | Out-Null",
```
Comment (Ash258) [inline bucket/mp3directcut.json]:
```suggestion
            "    Set-Content \"$dir\\mp3DirectCut.ini\" @('[mp3DirectCut]', \"version=$version\") -Encoding Ascii",
```

File: bucket/mp3directcut.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+            "} else {",
+            "    (Get-Content \"$persist_dir\\mp3DirectCut.ini\") -replace 'version=.+', \"version=$version\" | Set-Content \"$persist_dir\\mp3DirectCut.ini\"",
```
Comment (Ash258) [inline bucket/mp3directcut.json]:
```suggestion
            "    (Get-Content \"$persist_dir\\mp3DirectCut.ini\") -replace 'version=.+', \"version=$version\" | Set-Content \"$persist_dir\\mp3DirectCut.ini\" -Encoding Ascii",
```

**Pull request merged**